fecgov/demo-apps
# upgrade flask to v2.2.5

## Summary

Upgrade flask to v2.2.5 to resolve deployment issue when starting the flask app.

## Reference Ticket # https://github.com/pallets/flask/issues/4494

## Reviewers

One developer 

## Impacted areas of the application

demo-apps deployment on sandbox 

## How to test
- login to fec.gov sso account and choose `sandbox-fec` org
- follow [wiki](https://github.com/fecgov/demo-apps?tab=readme-ov-file#deploy-an-app-to-your-own-cloudgov-sandbox) to setup and deploy demo-apps to your sandbox space (deploy fails with jinja2 escape error)
- checkout this branch
- create a new virtualenv and activate it
- install requirements
- follow above wiki to deploy demo-apps
- access the flask app with url: https://demo-apps-fluent-alligator-cp.app.cloud.gov/
- screen shot for reference:
<img width="486" alt="Screen Shot 2024-04-10 at 11 11 20 AM" src="https://github.com/fecgov/demo-apps/assets/11650355/db5f0e33-d0ff-4ad5-9fc4-016fc77cdab3">

- verify app status on cloud.gov dashboard:
<img width="1062" alt="Screen Shot 2024-04-11 at 11 06 05 AM" src="https://github.com/fecgov/demo-apps/assets/11650355/4a315669-570f-4dff-8ab5-1d0ea0207256">
